src/logic/constants/index.js

```diff
@@ -0,0 +1,3 @@
+import ApplicationPackage from '../../../package.json';
+
+export const application_version = ApplicationPackage.version;
```

Comment:
```suggestion
export const APPLICATION_VERSION = ApplicationPackage.version;
```
This is screaming snake case, your variant was snake case.